src/mono/wasm/wasm.targets

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <WasmPInvokeModules Include="libSystem.Native"/>
+    <WasmPInvokeModules Include="__Internal"/>
```

Comment:
"QCall" https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/Interop/Interop.Libraries.cs#L9
and all the changes in `native-library.c` are not needed any more